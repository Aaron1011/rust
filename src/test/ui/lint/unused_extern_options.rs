// aux-crate:unused_crate=unused_crate.rs

#![deny(unused_extern_options)]

fn main() {}
