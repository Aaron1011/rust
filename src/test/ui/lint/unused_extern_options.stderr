error: crate `unused_crate` is unused in crate `unused_extern_options`
   |
note: the lint level is defined here
  --> $DIR/unused_extern_options.rs:3:9
   |
LL | #![deny(unused_extern_options)]
   |         ^^^^^^^^^^^^^^^^^^^^^
   = help: try removing `unused_crate` from your `Cargo.toml`

error: aborting due to previous error

